Python/jit.c

```diff
@@ -390,66 +406,128 @@ patch_x86_64_32rx(unsigned char *location, uint64_t value)
     patch_32r(location, value);
 }
 
+void patch_aarch64_trampoline(unsigned char *location, int ordinal, CompileState *state);
+
 #include "jit_stencils.h"
 
+#if defined(__aarch64__) || defined(_M_ARM64)
+    #define TRAMPOLINE_SIZE 16
+#else
+    #define TRAMPOLINE_SIZE 0
+#endif
```

Comment:
It's only used on this platform, so we can define it to 16 unconditionally:

```suggestion
#define TRAMPOLINE_SIZE_AARCH64 16
```

Reply:
You are right but the aim was to keep the code generic in case other platforms want to use this mechanism as well.

Reply:
I see, I missed that it was also being used in `_PyJIT_Compile`. You can go ahead and put it back the way it was!

Reply:
Done!